mining.md

```diff
@@ -132,7 +132,7 @@ func VerifyBlock(blk Block) {
 
 	// 3. Verify miner has not been slashed and is still valid miner
 	curStorageMarket := LoadStorageMarket(blk.State)
-	if !curStorageMarket.IsMiner(blk.Miner) {
+	if !curStorageMarket.IsMiner(blk.Miner) && !blk.Miner.ProvingPeriodEnd <= blk.height {
```

Comment:
duplicate checks here to l. 260. Which should be kept? Both checks should happen together in any case. 

May be worth amending `isMiner` if it's not used elsewhere... @dignifiedquire ?

Reply:
- nit: `blk.Miner.ProvingPeriodEnd > blk.height` is easier to read
- I am not sure what exactly this protects against, could you elaborate?